packages/core/src/cli/inspect.ts

```diff
@@ -10,6 +10,7 @@ export async function inspect(
   const environments = await composeRsbuildEnvironments(config);
   const rsbuildInstance = await createRsbuild({
     rsbuildConfig: {
+      plugins: config.plugins,
```

Comment:
missing `mf dev` commands in `packages/core/src/cli/mf.ts`?

Reply:
Nice catch, this has been fixed